DFC.ServiceTaxonomy.Neo4j/Services/NeoGraphDatabase.cs

```diff
@@ -56,12 +57,20 @@ public async Task<List<T>> Run<T>(IQuery<T> query)
 
         public async Task Run(params ICommand[] commands)
         {
-            IAsyncSession session = _driver.AsyncSession();
+            var executionTasks = _drivers.AllDrivers().Select(x => ExecuteTransaction(commands, x));
+            await Task.WhenAll(executionTasks);
+        }
+
+        private async Task ExecuteTransaction(ICommand[] commands, INeoDriver neoDriver)
+        {
+            IAsyncSession session = neoDriver.Driver.AsyncSession();
             try
             {
                 // transaction functions auto-retry
                 //todo: configure retry? timeout? etc.
 
+                _logger.LogInformation($"Executing Write commands to: {neoDriver.Uri}");
```

Comment:
Could this log just 'commands' rather than 'Write commands'? (Extra picky i know 😄!)